### PR TITLE
rhsso update 7.2.2.GA

### PR DESCRIPTION
**Pull request by fh-automatron**
Created automatically by jenkins

### CODE REVIEW

File: inventories/group_vars/all/manifest.yaml [hunk 1]
```diff
@@ -57,11 +57,11 @@ launcher_sso_template: https://raw.githubusercontent.com/jboss-container-images/
 
 #not currently used but is the version installed by the operator
 # rhsso is not an optional component
-rhsso_version: '7.2.0.GA'
+rhsso_version: '7.2.2'
```
Comment (mikenairn) [inline inventories/group_vars/all/manifest.yaml]: @philbrookes Does v1.2.2 of the keycloak operator install version 7.2.2 of keycloak? https://github.com/integr8ly/keycloak-operator/blob/master/pkg/keycloak/keycloak.go#L30

Comment (philbrookes) [reply inventories/group_vars/all/manifest.yaml]: it installs `7.2.2.GA`

Comment (mikenairn) [reply inventories/group_vars/all/manifest.yaml]: Is it possible to have this value set to "7.2.2.GA"  in the operator? https://github.com/integr8ly/keycloak-operator/blob/master/pkg/keycloak/keycloak.go#L30 That way this value would get set properly

Comment (mikenairn) [reply inventories/group_vars/all/manifest.yaml]: That's if we actually care about the .GA on the end here. @maleck13 Think i might have asked you about this before.

Comment (philbrookes) [reply inventories/group_vars/all/manifest.yaml]: I don't think the operator actually uses that value anywhere.

Comment (philbrookes) [reply inventories/group_vars/all/manifest.yaml]: it's only used here: https://github.com/integr8ly/keycloak-operator/blob/9cbca1d734aeeebcb8e504ce672b21c34b641261/pkg/keycloak/phaseHandler.go#L55 and that `Status.Version` isn't used anywhere.

Comment (mikenairn) [reply inventories/group_vars/all/manifest.yaml]: rhsso_version isn't used in the installer either, so .... doesn't seem like in either case it matters what the value actually is so we can probably just leave it as 7.2.2 and merge this PR.

Comment (maleck13) [reply inventories/group_vars/all/manifest.yaml]: It is used. https://github.com/integr8ly/installation/blob/master/roles/customisation/tasks/main.yaml#L108

Comment (mikenairn) [reply inventories/group_vars/all/manifest.yaml]: OK. Does it need the .GA on the end for anything? Is that significant for whatever that is doing?

**Pull request merged**